src/parameterizations/vertical/MOM_set_diffusivity.F90

```diff
@@ -2180,19 +2184,20 @@ subroutine set_diffusivity_init(Time, G, GV, US, param_file, diag, CS, int_tide_
   call get_param(param_file, mdl, "KV", CS%Kv, &
                  "The background kinematic viscosity in the interior. "//&
                  "The molecular value, ~1e-6 m2 s-1, may be used.", &
-                 units="m2 s-1", scale=US%m2_s_to_Z2_T, fail_if_missing=.true.)
+                 units="m2 s-1", scale=GV%m2_s_to_HZ_T, fail_if_missing=.true.)
 
-  call get_param(param_file, mdl, "KD", CS%Kd, &
+  call get_param(param_file, mdl, "KD", Kd_z, &
                  "The background diapycnal diffusivity of density in the "//&
                  "interior. Zero or the molecular value, ~1e-7 m2 s-1, "//&
                  "may be used.", default=0.0, units="m2 s-1", scale=US%m2_s_to_Z2_T)
+  CS%Kd = (GV%m2_s_to_HZ_T*US%Z2_T_to_m2_s) * Kd_z
```

Comment:
Can this be replaced with `GV%Z_to_H * Kd_z`?

Reply:
Usually, but not in all cases because we introduced RHO_KV_CONVERT to use in place of RHO_0 for in the expression fro `GV%m2_s_to_HZ_T` in MOM_vertical_Grid.F90.  By default they are the same.  However, in a later commit on my development branch we reused RHO_KV_CONVERT to define `GV%m_to_H`, so this expression could have been replaced with `CS%Kd = (US%Z_to_m*GV%m_to_H) * Kd_z` and it would have been unchanged.